### DISCUSSION TITLE
Cannot resolve "Adonis/Lucid/Relations" namespace

### DISCUSSION THREAD

**Discussion by b23618** [adonisjs/core#2702] [Help] [answered]
I cannot auth error 'E_IOC_LOOKUP_FAILED: Cannot resolve "Adonis/Lucid/Relations" namespace from the IoC Container'

how to fixed ??

**Comment by bitkidd** [comment, marked as answer]
I guess you are trying to import something like `BelongsTo` from there, just import it from `Adonis/Lucid/Orm`
